.devcontainer/devcontainer.json

```
@@ -1,9 +1,7 @@
 {
   "name": "Mathlib4 dev container",
 
-  "build": {
-    "dockerfile": "Dockerfile"
-  },
+  "image": "leanprovercommunity/gitpod4",
```

```suggestion
  "image": "leanprovercommunity/gitpod4",
```
@pechersky, what should I be using here now?

---

ghcr.io/leanprover-community/mathlib4

---

Has someone ran the workflow via workflow dispatch yet?

---

> ghcr.io/leanprover-community/mathlib4 

Really? Looking at your PR, isn't there a workflow name at the end of the name too?

---

I've just triggered a run: https://github.com/leanprover-community/mathlib4/actions/runs/14915728318

---

We can remove the attestation step. There probably needs to be some type of permission set on the workflow / github token to make it work.

@yael yes just that, there's no workflow suffix. Which do you mean??

---

Maybe #24694 fixes the issue?

---

Update: [it does not](https://github.com/leanprover-community/mathlib4/actions/runs/14920606809/job/41915081835).

---

Oh I meant https://github.com/leanprover-community/mathlib4/pull/23844/files#diff-7a41826c9e419fe38a2da72e1c873ad8ed7eff0ef7cf097c7a99905a261e481bR32 but looks I misread and this is just the repo name.

But now I am confused: There are three Dockerfiles to build: https://github.com/leanprover-community/mathlib4/tree/master/.docker and you're only building the second one. The first is useful for projects that don't depend on mathlib, and the third for projects with a blueprint

---

as a heads-up: I've started #24702 to fix things and it does build and publish the image. I'll see about the other 2 images and maybe it would be good to test the gitpod setup from this PR there

---

I had only set up the workflow for the gitpod container because that one was the one you had been pushing to dockerhub.

---

Ah 😅 I have been pushing all three: https://hub.docker.com/u/leanprovercommunity